ocw_import/api.py

```diff
@@ -135,8 +135,19 @@ def convert_data_to_content(filepath, data, website):  # pylint:disable=too-many
             parent, _ = WebsiteContent.objects.get_or_create(
                 website=website, text_id=str(uuid.UUID(parent_uid))
             )
+        # Assumes that s3 objects will be independently synced to the ocw-studio bucket via devops
+        file_location = content_json.get("file_location", None)
+        if file_location:
+            ocw_prefix = (
+                website.starter.config.get("root-url-path", "courses")
+                if website.starter
+                else "courses"
+            )
+            file_location = file_location.replace("coursemedia", ocw_prefix)
```

Comment:
Since we only want to replace a prefix here, can we check that `file_location` starts with `coursemedia`, and then only replace the one instance?  It's unlikely that `coursemedia` would be elsewhere in the URL, but if it is this code would replace it with `ocw_prefix`